ote/src/cljs/ote/app/controller/pre_notices.cljs

```diff
@@ -13,14 +13,12 @@
 (defn valid-notice? [notice]
   true)
 
-(declare ->ServerError ->RegionsResponse)
+(declare ->LoadPreNoticesResponse ->LoadPreNotice ->LoadPreNoticeResponse >ServerError ->RegionsResponse)
```

Comment:
>ServerError "-" missing

Reply:
Ok, this seems to be from conflict resolving